seed/java-sdk/exhaustive/no-custom-config/src/main/java/com/seed/exhaustive/resources/endpoints/container/ContainerClient.java

```diff
@@ -75,6 +83,44 @@ public List<String> getAndReturnListOfPrimitives(List<String> request, RequestOp
         }
     }
 
+    public CompletableFuture<List<String>> getAndReturnListOfPrimitivesAsync(List<String> request) {
+        return getAndReturnListOfPrimitivesAsync(request, null);
+    }
+
+    public CompletableFuture<List<String>> getAndReturnListOfPrimitivesAsync(
+            List<String> request, RequestOptions requestOptions) {
+
+        OkHttpClient client = clientOptions.httpClient();
+        if (requestOptions != null && requestOptions.getTimeout().isPresent()) {
+            client = clientOptions.httpClientWithTimeout(requestOptions);
+        }
+        CompletableFuture<List<String>> future = new CompletableFuture<>();
+        client.newCall(prepareGetAndReturnListOfPrimitives(request, requestOptions))
+                .enqueue(new Callback() {
+                    @Override
+                    public void onFailure(@NotNull Call call, @NotNull IOException e) {
+                        future.completeExceptionally(
+                                new SeedExhaustiveException("Network error executing HTTP request", e));
+                    }
+
+                    @Override
+                    public void onResponse(@NotNull Call call, @NotNull Response response) throws IOException {
+                        ResponseBody responseBody = response.body();
```

Comment:
I believe the `ResponseBody` is closeable so we would want to implement `try` with `resources`